kyaml/fn/runtime/runtimeutil/runtimeutil_test.go

```diff
@@ -1150,16 +1150,16 @@ metadata:
 `,
 			expectedFn: `
 container:
-    image: foo:v1.0.0
-    mounts:
-      - type: bind
```

Comment:
Why these list items have 2 space identations?

Reply:
The indent was 4, and I think '- ' counts as 2 spaces for the sake of indentation.